bcrs-ui/src/views/Home.vue

```diff
@@ -0,0 +1,18 @@
+<template>
+  <div class="home">
+    <img alt="Vue logo" src="../assets/logo.png">
+    <HelloWorld msg="Welcome to Your Vue.js App"/>
+  </div>
+</template>
+
+<script>
+import { Component, Vue } from 'vue-property-decorator'
+import HelloWorld from '@/components/HelloWorld.vue'
```

Comment:
Should we have a `@/components/index.ts`, as in other folders? Just a thought, not saying we should.

Reply:
Hmm, i thought about that too. For the sake of examples, i think we should! Agreed 👍